src/scripts.js

```diff
@@ -80,6 +80,24 @@ function generateUser() {
   findPantryInfo();
 }
 
+// POST FETCH REQUEST
+function changePantryIngredientAmount(userId, ingredientId, ingredientAmount) {
+  fetch("http://localhost:3001/api/v1/users", {
+    method: "POST",
+    body: JSON.stringify({
+      userID: userId,
+      ingredientID: ingredientId,
+      ingredientModification: ingredientAmount
+    }),
+    headers: {
+   'Content-Type': 'application/json'
+    }
+  })
+    .then(response => response.json())
+    .then(data => console.log(data))
+    .catch(error => console.log(error))
+}
```

Comment:
Great work with this function!  Let's figure out where in the DOM we would like to display our error handling and I think this would be done!